trappy/ftrace.py

```diff
@@ -76,6 +108,22 @@ def __init__(self, name="", normalize_time=True, scope="all",
             setattr(self, attr, trace_class)
             self.trace_classes.append(trace_class)
 
+        cache_path = _trace_cache_path(self.trace_path)
+        cache_good = False
+
+        if _check_trace_cache(self.trace_path):
+            cache_good = True
+            try:
+                # Read csv into frames
+                for trace_class in self.trace_classes:
+                    csv_file = cache_path + '/' + trace_class.unique_word[:-1] + '.csv'
+                    trace_class.read_csv(csv_file)
+            except:
+                cache_good = False
+
+        if cache_good:
+            return
```

Comment:
duplicated how?
data_frame will just be overwritten in that case
https://github.com/joelagnel/trappy/blob/97a29442e2c19bc5c09f73bee83f08d35406c907/trappy/base.py#L225
So I don't see how this is an issue